recipes/game-networking-sockets/all/conanfile.py

```diff
@@ -0,0 +1,114 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class GameNetworkingSocketsConan(ConanFile):
+    name = "game-networking-sockets"
+    description = "GameNetworkingSockets is a basic transport layer for games."
+    topics = ("networking", "game-development")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ValveSoftware/GameNetworkingSockets"
+    license = "BSD 3-Clause"
```

Comment:
```suggestion
    license = "BSD-3-Clause"
```

Use SPDX identifier https://spdx.org/licenses/